packages/swagger-zodios/src/types.ts

```diff
@@ -22,12 +22,20 @@ export type Options = {
      * @default `'barrel'`
      */
     exportType?: 'barrel' | 'barrelNamed' | false
+    /**
+     * Include `alias` in the generated endpoints, allowing for usage such as `apiClient.getUserById()`
+     *
+     * Defaults to the `operationId` in the OpenAPI file document for a given route. If no `operationId` is defined, it will be generated.
+     * @default `false`
+     */
+    includeOperationIdAsAlias: boolean | undefined
```

Comment:
can you make this optional: 
```
includeOperationIdAsAlias?: boolean
```

Reply:
Of course! updated in https://github.com/kubb-labs/kubb/pull/1101/commits/642d46d12b6e5981d4bb8cbaa1829ffd20721e16